tests/ui/trait-bounds/trait-bound-adt-issue-145611.rs

```diff
@@ -0,0 +1,8 @@
+trait LocalTrait {}
```

Comment:
please add a quick comment to explain this test. I would have to otherwise lookup the issue to figure out what it is testing